EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/GeneralMessageHandler/GeneralMessageHandler.cs

```diff
@@ -18,7 +20,7 @@ public async Task ProcessCommand(string userInput)
         // TODO: Consider the order of this once the scope of overall commands have been established. For efficency.
 
         var commandParams = userInput.Split(' ');
-        var command = commandParams!.First().ToLower();
+        var command = commandParams![0].ToLower();
```

Comment:
still not sure [0] instead of first?

Reply:
Okay so looking into it. The reason my compiler is suggesting the change is thanks to time complexity. Technically using LINQs First() is technically slower and also produces more overhead. If you want the Big O notation representations I will below. Since First() iterates over the entirety of the Collection before producing the result. its basically

array[0] = O(1)
array.First() = O(n)

Reply:
Ah interesting. I'm happy either or, first is more readable, but who doesnt know [0]